azure-spring-boot-starters/azure-mediaservices-spring-boot-starter/pom.xml

```diff
@@ -30,5 +30,9 @@
             <groupId>com.microsoft.azure</groupId>
             <artifactId>azure-media</artifactId>
         </dependency>
+        <dependency>
```

Comment:
why we want AI SDK here?

Reply:
because azure-media depends on ai sdk but our dependency marked ai as optional. Here to make sure ai sdk exist for azure-media, or may meet ClassNotFound from media starter.
Will double check it actually needed.

Reply:
seems no need, will remove this.